ansible/playbooks/AdoptOpenJDK_Windows_Playbook/README.md

```diff
@@ -0,0 +1,19 @@
+# Setting up Windows Machines
+
+There's a process to setting up Windows machines and getting them connected to Jenkins. If not followed, issues can occur with Jenkins workspaces (See: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1674).
+
+1) Log on to the Windows machine via RDP and run the `ConfigureRemotingForAnsible` commands listed in [main.yml](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml).
+
+2) Run the playbook on the machine, without skipping the 'adoptopenjdk' and 'jenkins' tags. (See [this](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/README.md) for more information).
+
+3) Login to the Jenkins user on the machine via RDP.
+
+4) On the machine, login to ci.adoptopenjdk.net, create a new Node and download/run the `slave-agent.jnlp`.
+
+5) Install the Jenkins agent as a service. This will require the Administrator credentials.
```

Comment:
Can you give the menu options for doing that?

Reply:
I think so - I believe it's just 'file > Install as a service' , but I'll double check that.

Reply:
Fixed :-)